piety-promoted-v1/modernized/02-chapter-01.adoc

```diff
@@ -298,21 +301,28 @@ Not only upon yourselves will you bring innocent blood, but upon the town,
 and the inhabitants thereof, and everywhere within this jurisdiction,
 that had the least hand therein.
 Therefore be instructed, you rulers of this land, and take warning betimes,
-and learn wisdom, before it be hid from your eyes.`'
+and learn wisdom, before it be hid from your eyes.
 
-_Written in the common jail the 19th of the Eighth month, 1659, in Boston,
+[.postscript]
```

Comment:
I originally put this as a [sscc], but then changed it to this. idk what's right to do, if anything!!_